cbsearchx/query_options.go

```diff
@@ -120,9 +120,7 @@ func (o *QueryOptions) encodeToJson() (json.RawMessage, error) {
 		}
 		m["ctl"] = encoder.EncodeField(control)
 	}
-	if o.Explain {
-		m["explain"] = encoder.EncodeField(o.Explain)
-	}
+	m["explain"] = encoder.EncodeField(o.Explain)
```

Comment:
Is this the right approach, or is it `!o.Explain { Encode(!xxx) }`, since the server default is false?

Reply:
I suppose that would save a small amount of bytes in the payload but otherwise no much benefit? I think this approach also insulates us to if the server decided to change the behaviour.

Reply:
+1